### PR TITLE
Ben/in app registration login ui

### PR DESCRIPTION
**Pull request by BenRettig2015**
Views for the In-App Registration + Log-in
 https://youtu.be/Twk2gD77Y6s

### CODE REVIEW

File: FightPandemics/ContentView.swift [hunk 1]
```diff
@@ -2,13 +2,34 @@
...
 //
+>>>>>>> a40a1a2ba53487f58584592f57bccc99f026086a
 
 import SwiftUI
 
 struct ContentView: View {
+<<<<<<< HEAD
+    init() {UINavigationBar
+        .appearance()
+        .titleTextAttributes = [.font: UIFont(name: "Verdana", size: 16)!]   }
```
Comment (Power186) [inline FightPandemics/ContentView.swift:18]: .font.. UIFont... not SwiftUI

File: FightPandemics/ViewTest.swift [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+//
+import SwiftUI
+struct Success: View {
+
+
+
+    init() {
+        
+
+        UINavigationBar.appearance().titleTextAttributes = [.font: UIFont(name: "Verdana", size: 16)!]
```
Comment (Power186) [inline FightPandemics/ViewTest.swift:15]: You can edit the navigation bar with modifiers in SwiftUI, not this archaic UIKit thing going on here, it's confusing seeing it mixed with SwiftUI

**Review by Power186 — CHANGES_REQUESTED** [review]
In Sign Up / Sign Up Form View, the logic and methods for authenticating a user should be encapsulated away from the view.  So all of those methods need to be put into a class, then add ObservableObject, then make instance in the SignUp/SignIn View with an @StateObject listener for that class, then use that State Object with the methods to show the right views.. This is MVVM pattern.  None of the code submitted here is MVVM.  I would be happy to explain or show further how to clean this code up if you like?  Or what do you think of this?

Other than that, nice work!

**Review by BenRettig2015 — COMMENTED** [review]
T

**Pull request closed without merging**